src/qibo/models/circuit.py

```diff
@@ -141,11 +141,9 @@ class Circuit:
             Defaults to ``False``.
         accelerators (dict, optional): Dictionary that maps device names to the number of times each
             device will be used. Defaults to ``None``.
-        wire_names (list or dict, optional): Names for qubit wires.
-            If ``None``, defaults to (``q0``, ``q1``... ``qn``).
+        wire_names (list, optional): Names for qubit wire names.
+            If ``None``, defaults to [``0``, ``1``, ..., ``nqubits - 1``].
             If ``list`` is passed, length of ``list`` must match ``nqubits``.
```

Comment:
This is not enough now, since `nqubits` became optional (so, if `None` you don't have anything to match).

Reply:
[a4f2092](https://github.com/qiboteam/qibo/pull/1500/commits/a4f2092708a9f15e066e28fdeb6da18eaaaa302f)